cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -183,7 +183,6 @@
       "session": {
         "runs": [
           {
-            "channel_uuid": "",
```

Comment:
Runs now longer have a channel - channel is on input (received on) and contact (preferred)